cdk/visit/lambda_code/log_visit/log_visit.py

```diff
@@ -164,65 +166,60 @@ def handle_log_visit_request(self, request, context):
             'Access-Control-Allow-Methods': 'OPTIONS,POST,GET'
         }
 
-        if (request is None):
+        def bad_request(body):
             return {
                 'headers': HEADERS,
                 'statusCode': 400,
-                'body': json.dumps({
-                    "Message": "Failed to provide parameters"
-                })
+                'body': json.dumps(body)
             }
 
+        # if no request is provided (should never be the case because of gateway evocation)
```

Comment:
Is evocation a real term APIGW uses? Do you mean invocation?

Reply:
Fixed!